### ISSUE TITLE
Cluster scheduling strategy-most-used

### ISSUE THREAD

**Issue by cisco66666** [open-webrtc-toolkit/owt-server#834]
1，Environment and platform：
master，ubuntu18.04，software，chrome-Version 87
2，Reproduce step：

scene 1
1,Two machines, 90 start all mcu modules, 51 start mcu webrtc module，
2,90 Configure node scheduling strategy in cluster-manager/cluster-manager.toml to most-used
Use the following command on the 51 machine: iperf -u -c (ip address or connectable network) -b 100M -t -1
nload 51 bitrate===100M
3, 51 Print out rt_load {Math.round(Math.max(rx_Mbps / max_scale, tx_Mbps / max_scale) * 1000) / 1000;} It is found that as long as the webrtc-agent is started separately, it will always be 0

To put it simply, if you start a webrtc-agent module machine (51) alone, rt_load will always be 0, if you start a machine with all modules (90), rt_load will always be greater than 0.
The webrtc process is always on the 90 machine, no matter what bitrate changes

3，Expected ：
The rt_load value is calculated correctly
